presto-main/src/main/resources/webapp/tableau/presto-connector.html

```diff
@@ -135,7 +135,7 @@
 
         function populateCatalogs() {
             var catalogClient = new StatementClient({
-                    'catalog': 'hive',
```

Comment:
Presto supports `SHOW CATALOGS` without specifying a catalog or schema. We shouldn't have to specify these here (though that will require changes in the JS `StatementClient`).